packages/adapter-vercel/files/edge.js

```diff
@@ -2,8 +2,35 @@ import { Server } from 'SERVER';
 import { manifest } from 'MANIFEST';
 
 const server = new Server(manifest);
+
 const initialized = server.init({
-	env: /** @type {Record<string, string>} */ (process.env)
+	env: /** @type {Record<string, string>} */ (process.env),
+	read: (file) => {
+		const controller = new AbortController();
+		const signal = controller.signal;
+
+		return new ReadableStream({
+			async start(controller) {
+				try {
+					const response = await fetch(manifest.appPath + file, { signal });
+					const reader = /** @type {ReadableStream} */ (response.body).getReader();
+
+					while (true) {
+						const { done, value } = await reader.read();
+						if (done) break;
+						controller.enqueue(value);
+					}
+
+					controller.close();
+				} catch (error) {
+					controller.error(error);
+				}
+			},
+			cancel(reason) {
+				controller.abort(reason);
+			}
+		});
+	}
```

Comment:
This should be a shared function imported from `@sveltejs/kit` (or `@sveltejs/kit/adapter-utils` or something). What is a good name for a function that synchronously returns a `ReadableStream` containing the body of an asynchronously fetched asset?

Reply:
`fetchFile`?